### DISCUSSION TITLE
Already started mission changes through eden editor

### DISCUSSION THREAD

**Discussion by mrtzkyle** [Vdauphin/HeartsAndMinds#1035] [Question and answer]
Hello, I would like to know how to add specific people as translators if possible. My mission is already started and a long way through.
Also while i'm here, how would I change the spawn on players and the logistics object spawn area with an already played map? thank you

**Comment by Vdauphin** [comment]
Hello,

> I would like to know how to add specific people as translators if possible. 

You mean interpreter?
In the init field of the unit add this: `this setVariable ["interpreter",true]`
Like it is done here: https://github.com/Vdauphin/HeartsAndMinds/blob/master/%3DBTC%3Dco%4030_Hearts_and_Minds.Altis/mission.sqm#L1059

> how would I change the spawn on players and the logistics object spawn area with an already played map

Just move them in the EDEN editor

Have fun!
